### DISCUSSION TITLE
Translations progress, contributors please read this!

### DISCUSSION THREAD

**Discussion by carlosperate** [mu-editor/mu#1931] [General]
First of all, **thank you!**

Any contributions to Mu are always very much appreciated, and translations are a very important part of making Mu accessible to people all over the world 🌎🌍🌏
https://madewith.mu/mu/users/2021/12/13/translate.html

## Purpose of this discussion thread

This page will try to list the state of all open translations to try to avoid any effort duplication.

The last release of Mu is beta.6, which hasn't had *any* of the translations updated, but since that release there might have been translations already done and merged into the project.

You can also use this thread to ask any questions you might have or let us know if you are planning to start any translation update.

## Translation instructions

In the docs :)
https://mu.readthedocs.io/en/latest/translations.html

## Translations awaiting review

If you don't have the time to do a full translation update, reviewing the submitted translations is also incredibly useful!

- None at the moment.

## Translations already updated

These translations have already been updated, but don't be discouraged! There might still be missing strings or some entries could be improved. In those cases please feel encouraged to open a new PR with the updates.

- Portugal Portuguese: https://github.com/mu-editor/mu/pull/1795
- Brazilian Portuguese: https://github.com/mu-editor/mu/pull/1924
- Slovak: https://github.com/mu-editor/mu/pull/1978 
- Russian: https://github.com/mu-editor/mu/pull/1886
- Japanese: https://github.com/mu-editor/mu/pull/1385
- Chinese (zh_CN): https://github.com/mu-editor/mu/pull/2033
- Dutch: https://github.com/mu-editor/mu/pull/2111
- Ukrainian: https://github.com/mu-editor/mu/pull/2261
- Czech: https://github.com/mu-editor/mu/pull/2446
- Thai: https://github.com/mu-editor/mu/pull/2447

## Translations not yet updated

All the others!

Mu already has some older translations in place, and you can see them in the [./mu/locale/](https://github.com/mu-editor/mu/tree/master/mu/locale) directory, so you don't have to start from zero, but if your language is not there yet we'd love to add it!

**Comment by merwok** [comment]
A question: is the name Mu the same as the greek letter µ?

(One translated string explains how to pronounce the name of the editor)

**Reply by diliboy88** [reply]
mu in greek is muscle.

**Reply by carlosperate** [reply]
That's a great question! @ntoll talks about this is https://github.com/mu-editor/mu/discussions/1447#discussioncomment-583919

**Reply by merwok** [reply]
Thanks 🙂   I’m taking from the answer that it would be correct if I noted in the French translation that the editor name is pronouced `mu` (like the letter) or `miou` (like in English).

**Reply by carlosperate** [reply]
Yeah, that sounds good!

**Comment by iamdbychkov** [comment]
Hello!
I've reviewed Russian translation of an editor and wanted to start working on Russian translation for a website but noticed that there are 2 pull requests with French and Chinese translations:

https://github.com/mu-editor/mu-editor.github.io/pull/49
https://github.com/mu-editor/mu-editor.github.io/pull/48

which are left without any activity since the end of 2018.

So here is the question, are they not merged because there were no reviewers or there is another reason to it?

**Reply by carlosperate** [reply]
Hi @iamdbychkov,

I haven't really kept an eye on the website issue tracker, but as all the maintainers are volunteers in their free I would probably guess the issue could have been a lack of time for review, the visibility of the PR (if we don't know a translation is coming), and unfortunately over time just falling through the cracks.

@ntoll do you have any insight?

**Comment by merwok** [comment]
I am slowly updating the french localization.  Many trivial conflicts are caused by the inclusion of line numbers in the comments; in my own projects I prefer to include only filename (good tradeoff to find the source and context easily).

**Comment by ghost** [comment]
Hi

I reviewed the Dutch translation and it's OK!
It can be added in the next version.

**Comment by TamilNeram** [comment]
Tamil Translation PR is waiting for so long https://github.com/mu-editor/mu/pull/2480
Anything to be done from my side for this?

**Reply by TamilNeram** [reply]
@carlosperate  FYA
